### ISSUE TITLE
Pending Approval: me.tylerbwong.gradle.metalava

### ISSUE THREAD

**Issue by tylerbwong** [gradle/plugin-portal-requests#69]
Hello! I recently submitted my plugin https://plugins.gradle.org/plugin/me.tylerbwong.gradle.metalava and it says "Pending Approval". I was wondering if someone could take a look and see if everything looked ok.

Also, will subsequent updates to the plugin also require manual approval?

**Comment by JLLeitschuh** [comment]
Sorry about that, just approved it.

**Comment by JLLeitschuh** [comment]
Subsequent versions will not require approval.

**Comment by tylerbwong** [comment]
Thanks for the approval!

I actually made a mistake on the artifact ID. It turns out the maven coordinates were incorrect. I'm having some trouble fixing it though. Any idea how I can make

```
me.tylerbwong.gradle:plugin:x.y.z
```

```
me.tylerbwong.gradle:metalava-gradle:x.y.z
```

instead?

I tried changing the `artifactId` in the `mavenCoordinates` block in https://github.com/tylerbwong/metalava-gradle/commit/79d531c51d716e534a0d9ab800778ed39e3e1be6, and it seems to have re-triggered the approval process, but I'm not sure if it actually changed the `artifactId`.
